crates/unwinder/src/exception_table.rs

```diff
@@ -173,6 +214,11 @@ pub struct ExceptionTable<'a> {
     callsites: &'a [U32Bytes<LittleEndian>],
     ranges: &'a [U32Bytes<LittleEndian>],
     tags: &'a [U32Bytes<LittleEndian>],
+    #[allow(
```

Comment:
```suggestion
    #[expect(
```

so that it turns into a warning (and CI failure) if we forget to remove this once it is actually used

Reply:
Done!